Reference List Customizations 1.tmPreferences

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>
+        meta.definition.function.ts,
+        meta.definition.method.ts
+    </string>
+    <key>settings</key>
+    <dict>
+        <key>showInIndexedReferenceList</key>
+        <string>0</string>
+    </dict>
+</dict>
+</plist>
```

Comment:
- [ ] Can't these two Reference List Customizations sit in the same file? (I dunno, just a question).
- [ ] I think the file should be called `Indexed Reference List.tmPreferences` to be in line with other packages
- [ ] I kinda like having that newline at the end

Reply:
I can't see any way to make them the same file in the plist format. They have to be loaded in a specific order which is why I used numbers, however I can re-name to `Exclusions` and `Inclusions` still keeping the order.

Ok re-named on my laptop.

Added newlines on my laptop.

- Have done the changes in an airport without wifi and replying on my phone... so won't be till later that I can actually push